imaging/bragg-edge-imaging-2D.ipynb

```diff
@@ -228,21 +227,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# exclude-from-export\n",
-    "fig1, ax1 = plt.subplots()\n",
-    "plot(sc.sum(sc.sum(ds[\"reference\"], 'x'), 'y'), ax=ax1)\n",
-    "for i in range(frames.sizes[\"frame\"]):\n",
-    "    ax1.axvspan(sc.mean(sc.mean(frames[\"time_min\"][\"frame\", i], 'x'), 'y').value,\n",
-    "                sc.mean(sc.mean(frames[\"time_max\"][\"frame\", i], 'x'), 'y').value,\n",
-    "                color=\"C{}\".format(i), alpha=0.3)\n",
-    "fig1"
+    "wfm.plot.frames_before_stitching(data=ds['reference'], frames=frames, dim='t')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Extract the sections in the original data using value-based slicing and shift the coordinates:"
+    "We then use the `wfm.stitch` function to convert the time dimension to time-of-flight.\n",
```

Comment:
We don't really explain this. We could now link in the wfm introduction notebook if we wanted users to understand what the procedure for this part of the problem would be like.